src/test/java/com/programmers/springweekly/repository/customer/JdbcTemplateCustomerRepositoryTest.java

```diff
@@ -0,0 +1,154 @@
+package com.programmers.springweekly.repository.customer;
+
+import com.programmers.springweekly.domain.customer.Customer;
+import com.programmers.springweekly.domain.customer.CustomerType;
+import com.programmers.springweekly.dto.CustomerUpdateDto;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ActiveProfiles("test")
+@SpringBootTest
+class JdbcTemplateCustomerRepositoryTest {
```

Comment:
해당 Repository 테스트에 필요한 bean들만 띄워보세요~!

Reply:
JdbcTemplateCustomerRepository.class를 주고 실행시켜보았으나 DataSource.class 의존성 문제로 계속 에러가나서 아래와 같이 DataSource도 주입 시켰는데 `No qualifying bean of type 'javax.sql.DataSource' available` 이런 에러가 나네요,,!
어떻게 처리해야할지 감이 안잡힙니다 😢 

```suggestion
@SpringBootTest(classes = {JdbcTemplateCustomerRepository.class, DataSource.class})
class JdbcTemplateCustomerRepositoryTest {
```

Reply:
DataSource를 등록하기 위해 또다른 bean을 등록해줘야하는것 같은데요,
일단 이 부분은 좀 복잡해질 수 있으니 넘어가시죠~ 제가 삽질 대신 해볼게요!